lib/ember-plugins.js

```diff
@@ -46,19 +46,22 @@ function _getDebugMacroPlugins() {
     ],
   ];
 }
-function _emberVersionRequiresModulesAPIPolyfill() {
-  // once a version of Ember ships with the
-  // emberjs/rfcs#176 modules natively this will
-  // be updated to detect that and return false
-  return true;
+function _emberVersionRequiresModulesAPIPolyfill(appRoot) {
```

Comment:
Can we use this version in both cases (essentially replacing the changes in `lib/babel-options-util.js`)?

Reply:
So just to clarify, you're saying that we should use the `resolvePackagePath` way of `ember-source` detection in the `babel-options-util.js`, rather than use `VersionChecker`?

If that is correct, I tried that in an earlier iteration but `resolvePackagePath` throws the following error when running `yarn test`:

```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type object
      at validateString (internal/validators.js:112:11)
      at Object.resolve (path.js:980:7)
      at resolvePackagePath (node_modules/resolve-package-path/lib/resolve-package-path.js:221:30)
      at resolvePackagePath (node_modules/resolve-package-path/index.js:86:19)
      at _emberVersionRequiresModulesAPIPolyfill (lib/babel-options-util.js:108:21)
      at _getEmberModulesAPIPolyfill (lib/babel-options-util.js:137:7)
      at getBabelOptions (lib/get-babel-options.js:55:7)
      at Class.buildBabelOptions (index.js:49:12)
      at Class.transpileTree (index.js:131:49)
      at Context.<anonymous> (node-tests/addon-test.js:1848:26)
      at Generator.next (<anonymous>)
      at onFulfilled (node_modules/co/index.js:65:19)
```

I then noticed we use `VersionChecker` approach throughout `babel-options-util.js` so went with that.

Reply:
In the case that failed were you passing a string in? Seems odd...